### PR TITLE
Feature: setup project

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

File: jsdoc.config.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+					"label": "Github",
+					"href": "https://github.com/ditrit/terrator-plugin#README.md"
+				}
+			]
+		}
+	},
+	"opts": {
+		"template": "node_modules/better-docs"
+	}
+}
```
Comment (justinehell) [inline jsdoc.config.json]: bug indent ?

File: tests/unit/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import Plugin from 'src/index';
+
+describe('Test index of project', () => {
+  it('Index should return GitlabPlugin', () => {
```
Comment (justinehell) [inline tests/unit/index.spec.js]:
```suggestion
  it('should return GitlabPlugin', () => {
```

File: tests/unit/parser/GitlabParser.spec.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import GitlabParser from 'src/parser/GitlabParser';
+import {
+  DefaultData,
+  FileInformation,
+} from 'leto-modelizer-plugin-core';
+
+describe('Test GitlabParser', () => {
+  describe('Test methods', () => {
+    describe('Test method: isParsable', () => {
+      it('Should return false', () => {
```
Comment (justinehell) [inline tests/unit/parser/GitlabParser.spec.js]: Should -> should

File: tests/unit/parser/GitlabParser.spec.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    describe('Test method: getModels', () => {
+      it('should return an empty array without parameter', () => {
+        const parser = new GitlabParser();
+
+        expect(parser.getModels()).toEqual([]);
+      });
+    });
+
+    describe('Test method: parse', () => {
+      it('Test default parse', () => {
```
Comment (justinehell) [inline tests/unit/parser/GitlabParser.spec.js]: should set pluginData components and parseErrors to empty array

File: tests/unit/render/GitlabRenderer.spec.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import GitlabRender from 'src/render/GitlabRenderer';
+import { DefaultData } from 'leto-modelizer-plugin-core';
+
+describe('Test GitlabRenderer', () => {
+  describe('Test method: renderFiles', () => {
+    it('Should render container', () => {
```
Comment (justinehell) [inline tests/unit/render/GitlabRenderer.spec.js]: should render an empty array

**Review by justinehell — CHANGES_REQUESTED** [review]

File: jsdoc.config.json (hunk 1 above)
Comment (Zorin95670) [reply jsdoc.config.json]: done

File: tests/unit/index.spec.js (hunk 2 above)
Comment (Zorin95670) [reply tests/unit/index.spec.js]: done

File: tests/unit/parser/GitlabParser.spec.js (hunk 3 above)
Comment (Zorin95670) [reply tests/unit/parser/GitlabParser.spec.js]: done

File: tests/unit/parser/GitlabParser.spec.js (hunk 4 above)
Comment (Zorin95670) [reply tests/unit/parser/GitlabParser.spec.js]: done

**Review by justinehell — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (k-y0u) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: improve what ?

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+ **Scenario** | `[the name for the behavior that will be described]`
+ ------------ | -------------------------------------------------
+ **Given**    | `[the beginning state of the scenario]`
+ **When**     | `[specific action that the user makes]`
+ **Then**     | `[the outcome of the action in “When”]`
+ **And**      | `[used to continue any of three previous statements]`
+
+### Constraints
+
+`[Put all others constraints here, like list of acceptances values or other]`
```
Comment (k-y0u) [inline .github/ISSUE_TEMPLATE/user_story.md:27]: acceptance values

File: src/metadata/GitlabMetadata.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import { DefaultMetadata } from 'leto-modelizer-plugin-core';
+
+/**
+ * Class to validate and retrieve components definitions from Gitlab metadata.
```
Comment (k-y0u) [inline src/metadata/GitlabMetadata.js]: component definitions

File: src/metadata/GitlabMetadata.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+ */
+class GitlabMetadata extends DefaultMetadata {
+  /**
+   * Validate the provided metadata with a schemas.
```
Comment (k-y0u) [inline src/metadata/GitlabMetadata.js]: schema

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 6 above)
Comment (Zorin95670) [reply .github/ISSUE_TEMPLATE/bug_report.md:3]: no modification, I will update all other issue template in another task.

File: .github/ISSUE_TEMPLATE/user_story.md (hunk 7 above)
Comment (Zorin95670) [reply .github/ISSUE_TEMPLATE/user_story.md:27]: no modification, I will update all other issue template in another task.

File: src/metadata/GitlabMetadata.js (hunk 8 above)
Comment (Zorin95670) [reply src/metadata/GitlabMetadata.js]: done

**Review by justinehell — APPROVED** [review]

**Review by k-y0u — APPROVED** [review]

**Pull request merged**